docs/topics/testing/advanced.txt

```diff
@@ -781,6 +781,41 @@ utility methods in the ``django.test.utils`` module.
     configuration that need to be reversed. It's the return value of the
     :meth:`setup_databases` method.
 
+.. function:: isolate_apps(*app_labels, attr_name=None, kwarg_name=None)
+
+    Register the models defined within a wrapped context into their own
+    isolated app registry. This functionality is useful when creating a
```

Comment:
link to app registry class and mention it by name, `Apps`, as per the existing contributing docs description

Reply:
👍